tests/functional/lt_test_rev_r_mem.c

```diff
@@ -15,13 +15,16 @@
 #include "lt_random.h"
 #include "string.h"
 
+/** @brief Maximal possible size of UDATA slot in User R-Memory accross all Application FWs. */
+#define R_MEM_DATA_SIZE_MAX 475
```

Comment:
Shouldn't we define this as a public macro? The use case would be defining an array with largest possible size to read data. Also, we could use this in lt_l3_api_structs, instead of using literal.